3.Tests/UI/appsettings.json

```diff
@@ -4,6 +4,8 @@
     "ProjectUri": "projects.json",
     "ItemUri": "items.json",
     "UserUri": "user.json",
+    "UserPutUri": "user/0.json",
```

Comment:
is the uri always going to be "0.json" to update user data? could it change to another number, say "15.json"?

Reply:
I though that it would only work with 0 but I just checked on the API and it works with any number. It basically updates the information of the authenticated user no matter the number